agent/__init__.py

```diff
@@ -66,7 +67,7 @@ def is_bootstrapping():
     # Create path if it doesn't exist
     if not os.path.isdir(CERT_PATH):
         os.makedirs(CERT_PATH)
-        os.chmod(CERT_PATH, 0o700)
+        os.chmod(CERT_PATH, 0o711)
```

Comment:
I think we should chmod these directories unconditionally. Because if we upgrade the older agent these directories will be present, but their mode will be 700.

Reply:
agreed